### PR TITLE
IT : Cleaning and Replacing

### PR DESCRIPTION
**Pull request by BellezaEmporium**
- Replaced a few links
- Kept adaptive links, or replaced chunklists with adaptive *when possible*
- Moved some channels
- Deleted duplicate channels

### CODE REVIEW

File: channels/it.m3u [hunk 1]
```diff
@@ -18,11 +18,11 @@ https://5f204aff97bee.streamlock.net/AltoAdigeTV/livestream/playlist.m3u8
 #EXTINF:-1 tvg-id="AntennaTreVeneto.it" tvg-country="IT" tvg-language="" tvg-logo="" group-title="",Antenna Tre Veneto (480p)
 https://59d8c0cee6f3d.streamlock.net/antennatreveneto/antennatreveneto.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="AuroraArte.it" tvg-country="IT" tvg-language="Italian" tvg-logo="https://i.imgur.com/Y1Yq3iV.png" group-title="",Aurora Arte (480p)
-https://59d7d6f47d7fc.streamlock.net/auroraarte/auroraarte/chunklist.m3u8
+https://59d7d6f47d7fc.streamlock.net/auroraarte/auroraarte/playlist.m3u8
 #EXTINF:-1 tvg-id="AzzurraTV.it" tvg-country="IT" tvg-language="Italian" tvg-logo="http://www.vcoazzurratv.it/images/1_tg_loghi/logo_vcoazzurratv_old_03.png" group-title="",Azzurra TV (576p)
 https://sb.top-ix.org/avtvlive/streaming/playlist.m3u8
 #EXTINF:-1 tvg-id="Bike.it" tvg-country="IT" tvg-language="" tvg-logo="https://i.imgur.com/wZh7aI8.png" group-title="",Bike (720p)
-https://backup.superstreaming.inaria.me/BikeSmartMobilityDTT/tracks-v2a1/playlist.m3u8
+http://backup.superstreaming.inaria.me/BikeSmartMobilityDTT/playlist.m3u8
```
Comment (freearhey) [inline channels/it.m3u:25]: https version also works
image

**Review by freearhey — APPROVED** [review]

File: channels/it.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/it.m3u:25]: Interesting, mine didn't like due to cert issue, that's why.

**Comment by Dum4G** [comment]
I can't moderate this repo either. Is it intentional?

**Comment by freearhey** [comment]
@Dum4G Try now.

**Review by Dum4G — APPROVED** [review]

**Pull request merged**